apps/db/lib/db/dataset_geographic_view.ex

```diff
@@ -3,7 +3,7 @@ defmodule DB.DatasetGeographicView do
   View to ease the geographic metadata of a Dataset
   """
   use Ecto.Schema
-  alias DB.Dataset
+  alias DB.{Dataset, Region}
```

Comment:
an include was missing? how was this working?

Reply:
No idea…